### ISSUE TITLE
Can't open project in external code editor on Linux

### ISSUE THREAD

**Issue by Wichamir** [MakovWait/godots#60]
I use the following settings to open VSCode:

![image](https://github.com/MakovWait/godots/assets/53086728/e0b8fcd4-87fe-4d5e-862e-99b74f6dcb42)

This works fine with vanilla Godot, but not with the one run by Godots.

![image](https://github.com/MakovWait/godots/assets/53086728/21b5cca7-4d9c-40bb-ac67-c6feb491eb3b)

**Comment by wokeraccoon** [comment]
Are you using the Flatpak version of Godots? I've noticed that I was having similar issues when trying to use VSCode with Godot Editors that were installed using Godots, and noticed that the Editor was having similar behavior as if I was using the Flatpak version of Godot, [so I used the instructions here to make VSCode work with Godot](https://github.com/flathub/org.godotengine.Godot#using-an-external-script-editor)

tl;dr: try using these exec path and exec flags if you're using the Flatpak version:

`Exec Path:  flatpak-spawn`
`Exec Flags: --host code --reuse-window {project} --goto {file}:{line}:{col}`

**Comment by MakovWait** [comment]
Have you resolved the issue using the @wokeraccoon solution?

**Comment by Wichamir** [comment]
Yes, it was in fact a problem with the Flatpak version. Using `flatpak-spawn` fixes the issue.
